dappnode_package.json

```
@@ -43,5 +43,11 @@
   "bugs": {
```

```suggestion
  "requirements": {
    "minimumDappnodeVersion": "0.2.62"
  },
  "bugs": {
```
need to bump minimum core to gnosis stakers UI which cant be until 0.2.62 at a minimum, i cant comment on the actual part since it was collapsed in the review, same for the bumping upstream, sorry for weird format

---

it was a suggestion on code thats not visible here, not possible to get it, just suggesting minimum core be set to `0.2.62` since i assumed thats the minimum core version with gnosis stakersUI implemented not important

---

Oh right, nice catch